skycatalogs/objects/snana_object.py

```diff
@@ -65,7 +72,10 @@ def _flux_ratio(mag):
                 # nothing else to do
                 return flux
 
-            # interpolate corrections
+            # interpolate corrections if we can
```

Comment:
I would put a comment here that the SNANA files can be delivered with nans, and in that case, the uncorrected flux is being returned.  I'd also print a warning message that this being done instead of silently handling things this way.